src/actions/MatrixActionCreators.js

```diff
@@ -37,11 +36,10 @@ function createSyncAction(matrixClient, state, prevState) {
 }
 
 /**
- * An action creator that will map an account data matrix event to a
- * MatrixActions.accountData action.
+ * Create a MatrixActions.accountData action that represents a MatrixClient `accountData`
+ * matrix event.
```

Comment:
ditto

Reply:
(it would help my brain if you could give some examples of what `event_type` is likely to be, here)